examples/knative/powershell/kn-ps-mm-sync/README.md

```diff
@@ -0,0 +1,42 @@
+# kn-ps-mm-vrops
+This example is a function which when a host enters Maintenance Mode in vCenter makes call to vRealize Operations Manager to mark host Maintenance Mode state.
+
+# Step 1 - Build container image
+
+Create the container images and optionally push to an external registry like GitHub Container Registry.
+
+```bash
+docker build -t <your-container-repository>/kn-ps-mm-sync-image:1.0 .
+# docker buildx build --platform linux/amd64 -t ghcr.io/darrylcauldwell/kn-ps-mm-sync-image:1.0 .
+docker push <your-container-repository>/kn-ps-mm-sync-image:1.0
+# docker push ghcr.io/darrylcauldwell/kn-ps-mm-sync-image:1.0
+```
+
+# Step 2 - Define environment variables
+
+In order to make successful call to vRealize Operations Manager environmentally specific FQDN and credentials are required. The function depends on environment variables passed as a Kubernetes secret.
+
+```bash
+# Define environment variables as secret
+
+kubectl -n vmware-functions create secret generic kn-ps-mm-sync-secret \
```

Comment:
Similarly during deployment to K8s, we can streamlines the secrets processing by using a file and `--from-file` option. Please see https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/development/examples/knative/powercli/kn-pcli-tag for an example